apps/system-e2e/entrypoint.sh

```diff
@@ -14,24 +14,23 @@ if [[ "$*" =~ --project ]]; then
 fi
 
 export TEST_PROJECT TEST_ENVIRONMENT TEST_TYPE TEST_RESULTS_S3 TEST_FILTER
-
-DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+export PATH="./node_modules/.bin:$PATH"
 
 echo "Current test environment: ${TEST_ENVIRONMENT}"
 echo "Playwright args: $*"
 echo "Playwright project: $TEST_PROJECT"
-echo "Playwright version: $(yarn playwright --version)"
+echo "Playwright version: $(playwright --version)"
 
 TEST_EXIT_CODE=0
-yarn playwright test -c src --project="$TEST_PROJECT" "$@" || TEST_EXIT_CODE=$?
+playwright test -c src --output test-results "$@" || TEST_EXIT_CODE=$?
```

Comment:
Are you sure you want to remove `--project`? Do we really need the test output (via `--outptut`)?

Reply:
I can't get the outputs to go into `test-results` :shrug:

Reply:
Removed `--output` usage. There are no more projects, so this should be safe. If we're still somewhere passing in `--project` for the container, the `entrypoint.sh` parses it out for us (and, now, disregards).